platforms/android/build_sdk.py

```diff
@@ -303,7 +236,6 @@ def gather_results(self, engines):
     parser.add_argument('--sign_with', help="Certificate to sign the Manager apk")
     parser.add_argument('--build_doc', action="store_true", help="Build javadoc")
     parser.add_argument('--no_ccache', action="store_true", help="Do not use ccache during library build")
```

Comment:
Could you disable ccache by default? (maybe change no_ccache to enable_ccache)
ccache in not available by default on Linux
ccache is not available on Windows